backend/src/controllers/sendEmail.ts

```diff
@@ -0,0 +1,60 @@
+const cron = require("node-cron");
+const nodemailer = require("nodemailer");
+// const UserModel = require('./models/user'); // Import your user model
+
+const sendMail = {
+  mail: async (req: any, res: any) => {
+    cron.schedule("08 00 * * *", async () => {
+      try {
+        // const usersToEmail = await UserModel.find({ email: { $nin: ['excluded@email.com'] } });
+        const usersToEmail = [
+          "chkushwaha@bestpeers.com",
+          "prichourasiya@bestpeers.com",
+        ];
+
+        const transporter = nodemailer.createTransport({
+          service: "gmail",
+          auth: {
+            user: "chkushwaha@bestpeers.com",
+            pass: "ppbe cugb xjno msqi",
+          },
```

Comment:
please create a new gmail account like portal12@gmail.com and use it's password don't use your office gmail.
it's a sensible thing @chandraprakash9589